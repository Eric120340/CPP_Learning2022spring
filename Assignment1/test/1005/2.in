4 4 9259089
796692706 6537617 311235031 919047865 
872779571 439048753 101631062 35737373 
479564831 142838516 134959084 867977347 
11134948 140813237 408438364 889175519 
