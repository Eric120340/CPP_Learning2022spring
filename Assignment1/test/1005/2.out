69808708 599986165 606399854 59126410 
453034758 964566659 285951259 286874974 
64589515 280923845 713784532 350195772 
953636691 434878073 531989697 770289525 
